tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/SQS/AwsSqsCommon.cs

```diff
@@ -56,5 +59,28 @@ public static Scope CreateScope(Tracer tracer, string operation, out AwsSqsTags
             // or we couldn't populate it completely (some tags is better than no tags)
             return scope;
         }
+
+        public static string GetQueueName(string queueUrl)
+        {
+            if (queueUrl is null)
+            {
+                return queueUrl;
+            }
+
+            return queueUrl.Split('/').Last();
+        }
+
+        internal static string GetOperationName(Tracer tracer, string spanKind) => tracer.CurrentTraceSettings.Schema.Version switch
+        {
+            SchemaVersion.V0 => SqsRequestOperationName,
+            _ => GetOperationNameHelper(spanKind)
+        };
+
+        internal static string GetOperationNameHelper(string spanKind) => spanKind switch
+        {
+            SpanKinds.Consumer => $"{SnsOperationName}.process",
+            SpanKinds.Producer => $"{SnsOperationName}.send",
```

Comment:
nitpicking - I believe those are the default values for `tracer.CurrentTraceSettings.Schema.Messaging.GetInboundOperationName` and `tracer.CurrentTraceSettings.Schema.Messaging.GetOutboundOperationName`. So maybe it would be better to call them rather than duplicating, WDYT?

Reply:
I agree. I skipped this due to the fact that it required me to send the `tracer` instance in order to access these methods. Nonetheless, I just realized that in case of a `V2`, this would require a greater effort. Moving forward with the suggested approach.